apps/openmw/mwworld/worldimp.hpp

```diff
@@ -602,7 +602,7 @@ namespace MWWorld
              * @brief Cast the actual spell, should be called mid-animation
              * @param actor
              */
-            void castSpell (const MWWorld::Ptr& actor) override;
+            void castSpell (const MWWorld::Ptr& actor, bool isScriptedSpell=false) override;
```

Comment:
We haven't enforced that much in the past (we are kinda relaxed about naming anyway), but I am noticing this issue more frequently now, so maybe it is time to bring it up. The name component "is" is typically meant for a function that returns a bool. A bool variable should not contain the word is.